src/constant.js

```diff
@@ -0,0 +1,24 @@
+import gdscLogo from "./assets/images/logo_GDSC.png";
```

Comment:
Good use of a constant file! I would advocate putting this file into a `src/utils/` folder since we'll probably have more helper files like this to come.